config/locales/idv/es.yml

```diff
@@ -128,9 +128,11 @@ es:
         seguro. La necesitará si pierde su contraseña.
       phone:
         alert: Este número de teléfono <strong>debe ser</strong>
+        description: Verificamos los registros para asegurarnos de que eres quien
+          eres, y de que eres el propietario de tu cuenta.
```

Comment:
might use `poseedor` rather than `propietario` (the holder/bearer/possessor of the phone rather than the proprietor/owner/landlord of the phone), but I'm fine with this if we're doing a translation review longer-term (TBD).